tests/rados/test_data_migration_bw_pools.py

```diff
@@ -0,0 +1,145 @@
+Scenarios:
+    RE -> RE
+    RE -> EC
+    EC -> RE
+    EC -> EC
+Methods:
+    rados cppool
+    rados import/export
+"""
+import time
+
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.test_9281 import do_rados_get, do_rados_put
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw) -> int:
+    """
+    Test to copy data from one pool to another
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    pool_obj = PoolFunctions(node=cephadm)
+    client_node = ceph_cluster.get_nodes(role="client")[0]
+
+    pool_configs_path = "suites/pacific/rados/pool-migration-configurations.yaml"
+
+    with open(pool_configs_path, "r") as fd:
+        pool_configs = yaml.safe_load(fd)
+
+    pool_orig = pool_configs[config["pool-1-type"]][config["pool-1-conf"]]
+    pool_target = pool_configs[config["pool-2-type"]][config["pool-2-conf"]]
+    create_given_pool(rados_obj, pool_orig)
+    create_given_pool(rados_obj, pool_target)
+
+    # Writing objects with huge omap entries
+    if not pool_obj.fill_omap_entries(pool_name=pool_orig["pool_name"], obj_end=500):
+        log.error(f"Omap entries not generated on pool {pool_orig['pool_name']}")
+        return 1
+
+    do_rados_put(mon=client_node, pool=pool_orig["pool_name"], nobj=1000)
+
+    snapshots = []
+    for _ in range(5):
+        snap = pool_obj.create_pool_snap(pool_name=pool_orig["pool_name"])
+        if snap:
+            snapshots.append(snap)
+        else:
+            log.error("Could not create snapshot on the pool")
+            return 1
+
+    # Using cppool to copy contents b/w the pools
+    cmd = f"rados cppool {pool_orig['pool_name']} {pool_target['pool_name']}"
+    client_node.exec_command(sudo=True, cmd=cmd, long_running=True)
+
+    # Sleeping for 2 seconds after copy to perform get operations
+    time.sleep(2)
+
+    do_rados_get(client_node, pool_target["pool_name"], 1)
+
+    # Checking if the snapshots of pool was also copied
+    # Snapshots of pool should not be copied
+    for snap_name in snapshots:
+        if pool_obj.check_snap_exists(
+            snap_name=snap_name, pool_name=pool_target["pool_name"]
+        ):
+            log.error("Snapshot of pool exists")
+            return 1
+
+    # deleting the Target pool created after cppool
+    rados_obj.detete_pool(pool=pool_target["pool_name"])
+
+    # Creating new target pool to test import/export
+    create_given_pool(rados_obj, pool_target)
+
+    # Creating temp file to hold pool info
+    client_node.exec_command(
+        sudo=True,
+        cmd="touch /tmp/file",
+    )
```

Comment:
Anyone can touch a file in `/tmp` folder... is there a specific reason you want `root privileges` to perform it?

Reply:
Removing root permission.